### ISSUE TITLE
Quick evaluation results

### ISSUE THREAD

**Issue by aazaff** [UW-COSMOS/Cosmos#147]
Here is an assessment of COSMOS returned results on the geothermal dataset (bigram model) for the search terms "thermal conductivity", "geochemistry", and "porosity" WITH the permalink to each success/failure included. Is there an ideal place to put this information?

[table checks.xlsx](https://github.com/UW-COSMOS/Cosmos/files/6402059/table.checks.xlsx)

**Comment by cambro** [comment]
Right here is fine.

**Comment by cambro** [comment]
Side note:
Some of returns identified as "incorrect" here are actually correct returns (i.e., the returned object has the search term) but the returned object is in some way truncated. This is either because the table spans pages, something we do not handle currently or because of some other segmentation error. These should ideally be flagged as correct but incomplete.
Example: https://xdd.wisc.edu/set_visualizer/sets/geothermal/object/07b481b52a9000ac82d1ff121bf549ec0df62a43

https://xdd.wisc.edu/set_visualizer/sets/geothermal/object/07b481b52a9000ac82d1ff121bf549ec0df62a43
--




https://xdd.wisc.edu/set_visualizer/sets/geothermal/object/07b481b52a9000ac82d1ff121bf549ec0df62a43
--

**Comment by aazaff** [comment]
I agree. I told anna to be extra conservative in her determinations which was maybe too strict. I will see if she has time to go through them today and revise the numbers.
